server/api/models.py

```diff
@@ -332,3 +332,18 @@ def save(self, *args, **kwargs):
                 '%(value)s is not a valid user category action',
                 params={'value': self.user_category_action.lower()},
             )
+
+
+class DeclinedInvite(BaseInfo):
+    """User that declined invite Model defined."""
+
+    user = models.ForeignKey(AndelaUserProfile, on_delete=models.CASCADE)
+    event = models.ForeignKey(Event, on_delete=models.CASCADE)
+
+    class Meta:
+        ordering = ('-created_at',)
+        unique_together = ('user', 'event')
+
+    def __str__(self):
+        return "@{} declined attending event {}" .format(
+            self.user.slack_name, self.event.title)
```

Comment:
@andela-mmakinde good stuff!
Just a side thought, is there a way we could have done this using the already existing Attend model without breaking the interface? Say introduce a field that stores acceptance or declination.

Reply:
@tomipaul Yes it's very much possible. But personally, I'll rather have the concerns separated. And again if I were to go that route, then there be changes in the current way the attendance is structured. 
I'll want the team to weigh in on this though.

Reply:
Well done! @makindemayowa I support we discuss this during our week sync tomorrow.